scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
```

Comment:
:stop_sign: **Security Vulnerability**: SQL injection vulnerability pattern detected[^4]. Directly interpolating user input into SQL queries can lead to SQL injection attacks.
```suggestion
import { Pool } from 'pg'

const pool = new Pool()

async function getUserById(id: string) {
    const query = 'SELECT * FROM users WHERE id = $1'
    const values = [id]
    try {
        const result = await pool.query(query, values)
        return result.rows[0]
    } catch (err) {
        console.error('Error executing query', err)
        throw err
    }
}
```